src/state/slices/preferencesSlice/selectors.ts

```diff
@@ -12,6 +13,12 @@ export const selectFeatureFlag = createCachedSelector(
   (featureFlags, flag) => featureFlags[flag],
 )((_, flag: Flag): Flag => flag ?? 'undefined')
 
+export const selectIsAssetWatched = createCachedSelector(
+  (state: ReduxState) => state.preferences,
```

Comment:
So we don't referentially invalidate on the whole `preferences` slice, and rename `preferences` to `watchedAssets` below

```suggestion
  (state: ReduxState) => state.preferences.watchedAssets,
```

Reply:
Actually there's already `selectWatchedAssets` below - let's move it above and we can consume it instead